src/main/java/rtg/init/BiomeInit.java

```diff
@@ -227,6 +291,192 @@ private static void init_betteragriculture() {
+        }
+        if (BOPBiomes.overgrown_cliffs.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPOvergrownCliffs());
+        }
+        if (BOPBiomes.prairie.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPPrairie());
+        }
+        if (BOPBiomes.quagmire.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPQuagmire());
+        }
+        if (BOPBiomes.rainforest.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPRainforest());
+        }
+        if (BOPBiomes.redwood_forest.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPRedwoodForest());
+        }
+        if (BOPBiomes.sacred_springs.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPSacredSprings());
+        }
+        if (BOPBiomes.seasonal_forest.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPSeasonalForest());
+        }
+        if (BOPBiomes.shield.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPShield());
+        }
+        if (BOPBiomes.shrubland.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPShrubland());
+        }
+        if (BOPBiomes.snowy_coniferous_forest.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPSnowyConiferousForest());
+        }
+        if (BOPBiomes.snowy_forest.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPSnowyForest());
+        }
+        if (BOPBiomes.steppe.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPSteppe());
+        }
+        if (BOPBiomes.temperate_rainforest.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPTemperateRainforest());
+        }
+        if (BOPBiomes.tropical_island.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPTropicalIsland());
+        }
+        if (BOPBiomes.tropical_rainforest.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPTropicalRainforest());
+        }
+        if (BOPBiomes.tundra.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPTundra());
+        }
+        if (BOPBiomes.volcanic_island.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPVolcanicIsland());
+        }
+        if (BOPBiomes.wasteland.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPWasteland());
+        }
+        if (BOPBiomes.wetland.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPWetland());
+        }
+        if (BOPBiomes.woodland.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPWoodland());
+        }
+        if (BOPBiomes.xeric_shrubland.isPresent()) {
+            RTGAPI.RTG_BIOMES.addBiomes(new RealisticBiomeBOPXericShrubland());
+        }
```

Comment:
This is not a safe way to check if BoP biomes should be added.
A list of known `ResourceLocations` should be used and a check made if that ResourceLocation key exists in the biome registry should be made. This will protect against biomes being removed whenever Forstide feels like making breaking changes, which could be at any time.